core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -578,16 +584,22 @@ class LogManager(logDirs: Seq[File],
   }
 
   /**
-   * Make a checkpoint for all logs in provided directory.
-   */
-  private def checkpointLogRecoveryOffsetsInDir(dir: File): Unit = {
+    * Make a checkpoint for all logs in provided directory by default.
+    * The affectedLogs argument is an optimization that can be used if we only want to checkpoint a few partitions.
+    *
+    * @param dir the directory in which logs are checkpointed
+    * @param affectedLogs logs whose snapshots need to be cleaned
```

Comment:
Would it be better to change "logs whose snapshots need to be cleaned" to "logs whose snapshots need to be cleaned. If it's None, the snapshot for all logs in the directory will be cleaned."?